src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -205,12 +206,16 @@
        [toolbar-view view-id unsigned-transactions-count]
        [main-section view-id tabs]])))
 
-(defn transaction-details-header [{:keys [value date type]}]
+(defn- pretty-print-asset [symbol amount]
+  (case symbol
+    "ETH" (if amount (money/wei->str :eth amount) "...")))
+
+(defn transaction-details-header [{:keys [value date type symbol]}]
   [react/view {:style transactions.styles/transaction-details-header}
    [react/view {:style transactions.styles/transaction-details-header-icon}
     [list/item-icon (transaction-type->icon type)]]
    [react/view {:style transactions.styles/transaction-details-header-infos}
-    [react/text {:style transactions.styles/transaction-details-header-value} (str value " ETH")]
+    [react/text {:style transactions.styles/transaction-details-header-value} (pretty-print-asset symbol value)]
```

Comment:
because for example here `transactions.styles` is too noisy

Reply:
and `transaction-details-` too, so it could be `styles/header`, `styles/header-value`

Reply:
much easy to read

Reply:
I agree, I will shorten style names.